theme/lumo/vaadin-details-styles.html

```diff
@@ -98,17 +89,18 @@
       }
 
       :host([theme~="small"]) [part="summary"] {
-        height: var(--lumo-size-s);
-      }
-
-      :host([theme~="small"]) [part$="content"] {
-        font-size: var(--lumo-font-size-s);
+        padding-top: var(--lumo-space-xs);
+        padding-bottom: var(--lumo-space-xs);
       }
 
       :host([theme~="small"]) [part="toggle"] {
         margin-right: calc(var(--lumo-space-xs) / 2);
       }
 
+      :host([theme~="small"]) [part$="content"] {
+        font-size: var(--lumo-font-size-s);
```

Comment:
This felt a little controversial, but I think it’s probably what users want in most cases, so I thought it was fine after all.